sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/firestore/FirestoreOptions.java

```diff
@@ -45,4 +46,15 @@ public interface FirestoreOptions extends PipelineOptions {
    * @see com.google.cloud.firestore.FirestoreOptions.Builder#setEmulatorHost(java.lang.String)
    */
   void setEmulatorHost(String host);
+
+  /**
+   * The Firestore database ID to connect to. Note: named database is currently an internal feature
+   * in Firestore. Do not set this to anything other than "(default)".
+   */
+  @Description("Firestore database ID")
+  @Default.String("(default)")
+  String getFirestoreDb();
```

Comment:
not sure on the consistency across beam, but given this is in the FirestoreOptions already, I would align this closer to the Firestore API naming convention: getDatabaseId / setDatabaseId (maybe getDatabase/setDatabase, but the Id suffix aligns better with our API style)

Reply:
+1. We should use whatever naming convention that makes most sense for the underlying Data store for Beam options.

Reply:
pcostell@ and I talked about this offline and decided to keep this option as firestoreDb to avoid ambiguity and conflict with other GCP service's connectors when specified from command line (e.g. Spanner uses databaseId option in their IT tests).